crates/common/consensus/src/execution_engine/mod.rs

```diff
@@ -135,4 +138,31 @@ impl ExecutionEngine {
             .await?
             .to_result()
     }
+
+    pub async fn engine_new_payloadv3(
+        &self,
+        execution_payload: ExecutionPayloadV3,
+        expected_blob_versioned_hashes: Vec<B256>,
+        parent_beacon_block_root: B256,
+    ) -> anyhow::Result<PayloadStatusV1> {
+        let request_body = JsonRpcRequest {
+            id: 1,
+            jsonrpc: "2.0".to_string(),
+            method: "engine_newPayloadV3".to_string(),
+            params: vec![
+                serde_json::json!(execution_payload),
+                serde_json::json!(expected_blob_versioned_hashes),
+                serde_json::json!(parent_beacon_block_root),
```

Comment:
```suggestion
                json!(execution_payload),
                json!(expected_blob_versioned_hashes),
                json!(parent_beacon_block_root),
```

you should improve `json!()` at the top of the file instead of repetitively including the import. Do this for the other functions in this file as well.